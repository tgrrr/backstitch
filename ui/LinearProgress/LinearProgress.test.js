import React from 'react'
import renderer from 'react-test-renderer'
import { LinearProgress } from '.'

it('LinearProgress: default', () => {
    const component = renderer.create(
        <LinearProgress>
            <>
                LinearProgress
            </>
        </LinearProgress>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
