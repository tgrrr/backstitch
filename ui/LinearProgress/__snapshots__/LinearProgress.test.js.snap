// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LinearProgress: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Linearprogress "
    data-testid="Linearprogress"
  >
    <div
      className="MuiLinearProgress-root MuiLinearProgress-colorPrimary MuiLinearProgress-indeterminate"
      role="progressbar"
    >
      <div
        className="MuiLinearProgress-bar MuiLinearProgress-barColorPrimary MuiLinearProgress-bar1Indeterminate"
        style={Object {}}
      />
      <div
        className="MuiLinearProgress-bar MuiLinearProgress-bar2Indeterminate MuiLinearProgress-barColorPrimary"
        style={Object {}}
      />
    </div>
  </div>
</div>
`;
